website/content/docs/concepts/domain-model/aliases.mdx

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+page_title: Domain model - aliases
+description: |-
+  The anatomy of a Boundary alias
+---
+
+# Aliases
+
+An alias is a resource that lets you associate a string with a specific target.
+The string must be a globally unique value which follows DNS naming constraints.
+You can use the alias instead of the target ID when you run a command, and Boundary automatically substitutes the target resource that the alias references.
+
+## Attributes
+
+All alias types have the following attributes:
```

Comment:
Since there's just one type for the time being, is it worth just saying: 

Aliases have the following attributes: